CMakeLists.txt

```diff
@@ -27,19 +27,27 @@ if (NOT CMAKE_BUILD_TYPE)
         FORCE)
 endif()
 
-set(AEON_VERSION_MAJOR 1)
-set(AEON_VERSION_MINOR 3)
-set(AEON_VERSION_PATCH 3)
+# Set directory where the custom finders live
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+
+# Get actual aeon version number from sources
+include(AeonVersion)
 
-set(AEON_BUILD_NUMBER 0)
+AEON_GET_VERSION_LABEL()
+AEON_GET_CURRENT_HASH()
```

Comment:
Did you mean `AEON_GET_TAG_OF_CURRENT_HASH()` ?

Reply:
No, only one call will remain after the above change.